tests/unit/inference/test_inference.py

```diff
@@ -384,6 +384,47 @@ def test(
 
 
 @pytest.mark.seq_inference
+@pytest.mark.parametrize("model_w_task", [("EleutherAI/gpt-j-6B", "text-generation")], ids=["gpt-j"])
```

Comment:
@mrwyattii - what tests did you want to run here to avoid OOM issues?

Reply:
Let's go with a small model to keep test times to a minimum: `gpt2`